feat: Math syntax enabled default (#94)
vivliostyle/vfm: docs/vfm.md

**HTML**

> It also outputs `<script>` for processing MathJax in Vivliostyle if `math` is enabled. However, the math syntax or the `<math>` tag must actually exist.

Reviewer: この `<script>` は、Vivliostyleのためではなく、普通にブラウザで HTML を表示したときに、MathJax を有効にするためのものなので、 "in Vivliostyle" は削除するとよいです。

参照: https://github.com/vivliostyle/vfm/issues/37#issuecomment-813201639
> Vivliostyle Viewer はこの script タグの有無に関係なく常に MathJax をロード。この script タグは、Vivliostyle Viewer を通してではなく、普通の Web ページとしてこの HTML を閲覧するときのために必要。

また、 "However, the math syntax or the `<math>` tag must actually exist." という書き方だと、`math` が有効なときは、必ず数式が存在する必要がある、と誤解されます。そこで、次のように１文にまとめるのがよいと思います：

```suggestion
It also outputs `<script>` for processing MathJax if `math` is enabled and the math syntax or the `<math>` tag is actually existed.
```
